helm_deploy/values-test.yaml

```diff
@@ -19,7 +19,9 @@ generic-service:
     COMMUNITY_ACCOMMODATION_API_SERVICE_NAME: approved-premises
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE: 30000
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_DEADLINE: 30000
-
+    FLIPT_ENABLED: true
+    FLIPT_URL: 'https://feature-flags-dev.hmpps.service.justice.gov.uk/'
```

Comment:
so for now our `test` and `dev` environments will always have their flags toggled in unison?